server/disk_avail.go

```diff
@@ -11,8 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !windows
-// +build !openbsd
+//go:build !windows && !openbsd
+// +build !windows,!openbsd
```

Comment:
We need both?

Reply:
These only apply to [Go 1.17](https://golang.org/doc/go1.17),  we will need both to avoid `go fmt` failing on Go 1.17 and to support Go 1.16 which does not recognize the new syntax.